src/net/mod.rs

```diff
@@ -5,19 +5,17 @@ mod client;
 mod server;
 
 pub mod discovery;
-pub mod http_network;
 
-use crate::sync::{Arc, Mutex};
-pub use client::MpcHelperClient;
+pub use client::{HttpSendMessagesArgs, MpcHelperClient};
 #[cfg(feature = "self-signed-certs")]
 pub use server::tls_config_from_self_signed_cert;
-pub use server::{BindTarget, MessageSendMap, MpcHelperServer};
+pub use server::{BindTarget, MessageSendMap, MpcHelperServer, MpcHelperServerError};
 use std::collections::HashMap;
 use std::str::FromStr;
 
 use crate::{
     helpers::{network::ChannelId, MESSAGE_PAYLOAD_SIZE_BYTES},
-    net::server::MpcHelperServerError,
+    sync::{Arc, Mutex},
```

Comment:
you would want to rebase this on top of main branch

Reply:
i believe it is already. is there something in particular that should be different?

Reply:
ah, you're right. for some reason I saw it as std::sync